packages/db/prisma/schema.prisma

```diff
@@ -54,6 +55,13 @@ enum ProblemType {
   Code
   MCQ
 }
+enum Category {
+  Fundamentals
```

Comment:
There should be a saparate table called categories
Each track should have one to many to it. Single track can be part of two categories
Everything else looks good

Reply:
@hkirat 
I have made changes in #148 can you please review it?